diff --git a/node_modules/oas3-tools/dist/middleware/express.app.config.js b/node_modules/oas3-tools/dist/middleware/express.app.config.js
index 14672c1..ee070ab 100644
--- a/node_modules/oas3-tools/dist/middleware/express.app.config.js
+++ b/node_modules/oas3-tools/dist/middleware/express.app.config.js
@@ -24,13 +24,11 @@ class ExpressAppConfig {
         this.app.use(express.json());
         this.app.use(express.urlencoded({ extended: false }));
         this.app.use(cookieParser());
-        const swaggerUi = new swagger_ui_1.SwaggerUI(swaggerDoc, undefined);
-        this.app.use(swaggerUi.serveStaticContent());
+        // const swaggerUi = new swagger_ui_1.SwaggerUI(swaggerDoc, undefined);
+        // this.app.use(swaggerUi.serveStaticContent());
     }
-    addValidator() {
-        new express_openapi_validator_1.OpenApiValidator({
-            apiSpec: this.definitionPath,
-        })
+    addValidator(options) {
+        new express_openapi_validator_1.OpenApiValidator(options)
             .install(this.app)
             .then(() => {
             this.app.use(new swagger_parameters_1.SwaggerParameters().checkParameters());
